Sources/Error Handling/ErrorUtils.swift

```diff
@@ -320,11 +334,10 @@ enum ErrorUtils {
                 return backendError.asPurchasesError
             case let networkError as NetworkError:
                 return networkError.asPurchasesError
+            case let error as PublicError where error.domain == ErrorCode.errorDomain:
```

Comment:
This might be a stupid question but where do we define our `errorDomain`? I couldn't find it 😆

Reply:
Not a stupid question 😅 It’s Swift magic for `Swift.Error` conformances.
See also #1275.